WordPressShared/WordPressShared/Core/Views/WPNoResultsView.m

```diff
@@ -75,13 +75,16 @@ - (void)configureTitleLabel
 
 - (void)configureMessageLabel
 {
-    _messageLabel               = [[UILabel alloc] init];
-    _messageLabel.font          = [UIFont preferredFontForTextStyle:UIFontTextStyleSubheadline];
-    _messageLabel.textColor     = [WPStyleGuide allTAllShadeGrey];
-    _messageLabel.numberOfLines = 0;
-    _messageLabel.textAlignment = NSTextAlignmentCenter;
-    _messageLabel.adjustsFontForContentSizeCategory = YES;
-    [self addSubview:_messageLabel];
+    _messageTextView                 = [[UITextView alloc] init];
+    _messageTextView.font            = [UIFont preferredFontForTextStyle:UIFontTextStyleSubheadline];
+    _messageTextView.textColor       = [WPStyleGuide allTAllShadeGrey];
+    _messageTextView.backgroundColor = [UIColor clearColor];
+    _messageTextView.textAlignment   = NSTextAlignmentCenter;
+    _messageTextView.adjustsFontForContentSizeCategory = YES;
+    _messageTextView.editable = NO;
```

Comment:
the text field is still selectable, I think `selectable` should be `NO` as well.